app/models/compute_resource.rb

```diff
@@ -399,6 +399,26 @@ def normalize_vm_attrs(vm_attrs)
     vm_attrs
   end
 
+  def firmware_types
+    {
+      "automatic" => N_("Automatic"),
+      "bios" => N_("BIOS"),
+      "efi" => N_("EFI"),
+      "uefi_sb" => N_("UEFI Secure Boot"),
+    }.freeze
+  end
+
+  def firmware_mapping(firmware_type)
+    case firmware_type
+    when :uefi
+      'efi'
+    when :uefi_sb
+      'uefi_sb'
```

Comment:
I'd avoid abbreviations and write it out in full
```suggestion
    when :uefi_secure_boot
      'uefi_secure_boot'
```